tests/atlasapi/test_core_atlas.py

```diff
@@ -102,6 +102,22 @@ def test_data_from_coords(atlas, coords):
     )
 
 
+def test_data_from_coords_out_of_brain(
+    atlas, coords=(1, 1, 1), key_error_string="Outside atlas"
```

Comment:
What's the reason for `coords` and `key_error_string` to be arguments of the test function?

I think they should just be defined in the test function body?

Reply:
No reason. I think I usually do it so anything that may need to be changed is at the top of the function. Will move it into the function.